Managed_SCCA_Broker_(SCCAv2)/DESTROY.md

```diff
@@ -0,0 +1,71 @@
+# Guide to Destroy SCCAv2 Landing Zone Using destroy_lz.py
+
+### Prerequisites:
+1. Ensure all four sub-compartments (VDMS, VDSS, Logging, and Config) are present in the parent compartment
+2. Do not run terraform destroy before running the destroy_lz.py script
+   1. If you have already run terraform destroy, navigate to the vault resource in the cloud console and manually set the status to "active"
+
+### 1.	Navigate to the destroy_lz.py script:
+     /Managed_SCCA_Broker_(SCCAv2)/destroy_lz.py
+The script is in the same directory as this guide, titled "destroy_lz.py"
+
+### 2.	Download and move this script into your local landing zone directory:
+For example: /User/mylandingzone/scca_cis_multi_tenancy/destroy_lz.py
+
+### 3.	Run the destroy_lz.py script:
+Run this command in the command line (replace the information in the brackets with your compartment's information):
+
+```text
+python3 destroy_lz.py -c [YOUR LANDING ZONE COMPARTMENT NAME] -r [YOUR REGION KEY] -l [YOUR RESOURCE LABEL]  --template_name [TEMPLATE NAME (options: CHILD, PARENT, or WORKLOAD]
+```
+For example: python3 destroy_lz.py -c OCI-SCCA-LZ-CHILD-Home-RIC-miachildtest -r RIC -l miachildtest --template_name CHILD
+
+Note: You can find your region key in your named compartment
+
+Successful results of this script:
+1. The Central-Vault resource will be moved to the root compartment
+2. All resources except for compartments will be deleted
+
+### 4. Modify the terraform state file to omit the identity domains:
+Note: Terraform destroy will throw an error if this is not run (see known issues at the bottom)
+
+Run this command in the command line:
+```text
+terraform state rm 'module.scca_identity_domains[0]'
+```
+
+### 5.	Run terraform destroy:
+Run this command in the command line:
+```text
+terraform destroy
+```
+Successful results of this script:
+
+1. All compartments in this deployment will be deleted
+2. The Central-Vault resource will be marked for deletion in 30 days
+
+### 6.	OPTIONAL: Manually schedule vault deletion:
+The vault should already be marked for deletion in 30 days; however, you can schedule it for an earlier deletion by navigating to the cloud console and manually deleting your vault resource. It will give options on when to schedule it for deletion, but the earliest it can be deleted is 7 days of the current date.
```

Comment:
I think change "cloud console" to "OCI Console" and provide more precise instructions on manually cancelling the earlier deletion and changing the deletion date. Imagine your user hasn't deleted vaults before. Provide instructions on where to navigate, what buttons to click, etc.